archivebox/util.py

```diff
@@ -173,6 +173,20 @@ def download_url(url: str, timeout: int=None) -> str:
 
     return response.text
 
+@enforce_types
+def get_headers(url: str, timeout: int=None) -> str:
+    """Download the contents of a remote url and return the headers"""
+    from .config import TIMEOUT, CHECK_SSL_VALIDITY, WGET_USER_AGENT
+    timeout = timeout or TIMEOUT
+    response = requests.get(
```

Comment:
don't use a get request or it will download the entire response body! use a HEAD request to get only the headers

Reply:
My bad, didn't notice it when I reviewed. @afreydev can you correct it please?